inst/testdata/theo.saemix.tab

```diff
@@ -0,0 +1 @@
+../../data/theo.saemix.tab
```

Comment:
This is a link to the file in the data/ subdirectory. Having this link in inst/testdata causes the file to be installed with the package, so it can be accessed by the user or in a test script using
`system.file("testdata/theo.saemix.tab", package = "saemix")`.